tests/unit_tests/dft/include/compute_inplace_real_real.hpp

```diff
@@ -0,0 +1,127 @@
+/***************************************************************************
+*  Copyright (C) Codeplay Software Limited
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  For your convenience, a copy of the License has been included in this
+*  repository.
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+**************************************************************************/
+
+#ifndef ONEMKL_COMPUTE_INPLACE_REAL_REAL_HPP
+#define ONEMKL_COMPUTE_INPLACE_REAL_REAL_HPP
+
+/* Test is not implemented because currently there are no available dft implementations.
+ * This are stubs to make sure that dft::unimplemented exception is thrown */
```

Comment:
Maybe this was a small typo here? If so, the same typo is repeated in some of the other files below as well.

```suggestion
 * These are stubs to make sure that dft::unimplemented exception is thrown */
```